Firestore/Source/Remote/FSTRemoteStore.m

```diff
@@ -160,47 +160,47 @@ - (void)start {
   [self enableNetwork];
 }
 
-- (void)setOnlineStateToHealthy {
-  self.shouldWarnOffline = NO;
-  [self updateAndNotifyAboutOnlineState:FSTOnlineStateHealthy];
-}
+- (void)updateOnlineState:(FSTOnlineState)newState {
```

Comment:
Copy the comment from the JS port?

/**
  * Updates our OnlineState to the new state, updating local state
  * and notifying the onlineStateHandler as appropriate.
 */

Reply:
Woops, good catch. Thanks!